packages/middleware-flexible-checksums/src/flexibleChecksumsMiddleware.ts

```diff
@@ -78,14 +80,5 @@ export const flexibleChecksumsMiddleware =
       },
     });
 
-    const { requestValidationModeMember, responseAlgorithms } = middlewareConfig;
-    // @ts-ignore Element implicitly has an 'any' type for input[requestValidationModeMember]
-    if (requestValidationModeMember && input[requestValidationModeMember] === "ENABLED") {
-      await validateChecksumFromResponse(result.response as HttpResponse, {
-        config,
-        responseAlgorithms,
-      });
-    }
-
```

Comment:
the response portion of this middleware is moved to a separate middleware that goes after the deser stage. 

the plugin installs both of these new middleware to preserve functionality